docs/source/settings.rst

```diff
@@ -234,7 +234,13 @@ keepalive
 
 The number of seconds to wait for requests on a Keep-Alive connection.
 
-Generally set in the 1-5 seconds range.
+Generally set in the 1-5 seconds range for servers with direct connection
```

Comment:
Could you make this change in `gunicorn/config.py` instead and then run `make html` in `docs/` directory? (After running `make html` you will need to strip unrelated changes in `docs/source/settings.rst`.)